.travis.yml

```diff
@@ -8,7 +8,7 @@ matrix:
     - python: "3.6"
       env: TEST_CMD="./tests/mypy_selftest.py"
     - python: "3.5"
-      env: TEST_CMD="./tests/mypy_test.py --no-implicit-optional"
+      env: TEST_CMD="./tests/mypy_test.py --no-implicit-optional --no-site-packages"
```

Comment:
I don't think this is right, since mypy_test doesn't accept arbitrary arguments to pass to mypy. Let's just merge the one among this one and #2036 that passes Travis. :)